### DISCUSSION TITLE
Errors when installing MeTube

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#3204] [Help Needed] [answered]
Hi

I get an error when installing MeTube without defaults. i.e. when i choose to install with specifics, it causes an error:

`
 \Successfully installed certifi-2024.6.2 distlib-0.3.8 filelock-3.15.1 pipenv-2024.0.1 platformdirs-4.2.2 setuptools-70.0.0 virtualenv-20.26.2
 |Warning: Python 3.11 was not found on your system...
Neither 'pyenv' nor 'asdf' could be found to install Python.
You can specify specific versions of Python with:
$ pipenv --python path/to/python

[ERROR] in line 61: exit code 0: while executing command $STD pipenv install
`

However, when i install with the defaults, it works. But then it will use dhcp, which is no good for a server.

Any ideas?
thanks

**Comment by Mellowlynx** [comment]
Using DHCP is fine, as long as the server is up, the IP lease should renew.
And most routers have the option to fix an IP to a mac address, so it can be offline for a while before losing its 'reservation'.
So yes, it's better to have a static IP for servers, but fixing the IP to the mac address in your router is also fine and will have the same effect.

**Comment by tteck** [comment]
What Linux distribution are you choosing when selecting advanced settings?

**Comment by ghost** [comment]
I've tried with Debian and ubuntu, both don't work. I tired both debian versions, and i all the ubuntu versions. they work without a hitch when i use dhcp, but as sooon as I customise the install, it fails

**Reply by tteck** [reply, marked as answer]
Use Debian 12 only.

**Reply by tteck** [reply]
You must be entering something incorrectly when using advanced settings.

**Reply by ghost** [reply]
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: ********
Container ID: 114
Using Hostname: metube
Using Disk Size: 10
Allocated Cores: 1
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: 192.168.1.114/24
Using Gateway IP Address: 192.168.1.1
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: 192.168.1.109
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: yes

**Reply by ghost** [reply]
every other script i've used has worked with the same settings

**Reply by ghost** [reply]
it just worked with deb 12. I might not have installed it with debian 12 then, i think. I swear i did, but maybe i didn't. Thanks for the help
